pybossa/api/__init__.py

```diff
@@ -440,8 +440,10 @@ def task_gold(project_id=None):
         return abort(401)
 
     project = project_repo.get(project_id)
-    if project is None or not(current_user.admin
-        or current_user.id in project.owners_ids):
+
+    # Allow project owner, sub-admin co-owners, and admins to update Gold tasks.
+    isGoldAccess = (current_user.subadmin and current_user.id in project.owners_ids) or current_user.admin
```

Comment:
can we use `is_gold_access` instead?

Reply:
+1

Reply:
Fixed via https://github.com/bloomberg/pybossa/pull/252/commits/1698ac9923f9aad99feea15a3c8b30542fad756e